service/src/main/java/com/ge/predix/acs/attribute/cache/AttributeCacheFactory.java

```diff
@@ -1,17 +1,45 @@
 package com.ge.predix.acs.attribute.cache;
 
+import org.apache.commons.lang.BooleanUtils;
+import org.springframework.data.redis.core.RedisTemplate;
+
 // CHECKSTYLE:OFF: FinalClass
 public class AttributeCacheFactory {
 
+    private static final String SPRING_PROFILES = System.getenv("SPRING_PROFILES_ACTIVE");
```

Comment:
Problem is that this isn't a bean, so I don't think I can auto wire anything in here.